app/lib/alimonitor-services/BookkeepingService.js

```diff
@@ -141,6 +141,10 @@ class BookkeepingService extends AbstractServiceSynchronizer {
 
     metaDataHandler(requestJsonResult) {
         const { page } = requestJsonResult['meta'];
+        if (!page || !page['pageCount']) {
+            this.logger.error(`No page meta date from bookkeping: page=${JSON.stringify(page)}`);
```

Comment:
I believe that this message would sound more naturally like this: `No metadata found in Bookkeeping for the requested page: ${JSON.stringify(requestJsonResult)}`. It also provides more information in case of the null page (`!page`).